app/models/application_form.rb

```diff
@@ -277,7 +277,10 @@ def nationalities
   def full_address
     if international?
       [
-        international_address,
+        address_line1,
+        address_line2,
+        address_line3,
+        address_line4,
```

Comment:
Why is postcode not included here? It was on the form for international addresses but are we removing it now?

Reply:
In the discussion i had with Duncan, we said we would just use 4 inputs with no labels for international addresses as we weren't sure what shape the data would come back in. Just applying the changes now.

Reply:
![image](https://user-images.githubusercontent.com/42515961/107209786-2f86c000-69fb-11eb-88cc-7e724c7c6ba5.png)